packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/execute_event/v0/mod.rs

```diff
@@ -54,15 +59,17 @@ where
             ExecutionEvent::PaidFromAssetLockDriveEvent {
                 identity,
                 operations,
+                signature_verifications,
```

Comment:
No, this should be something like "validation_costs"

Reply:
Fixed. Renamed to `validation_operations`